pkg/controller/nodes/task/handler.go

```diff
@@ -541,6 +541,17 @@ func (t Handler) Handle(ctx context.Context, nCtx handler.NodeExecutionContext)
 				logger.Errorf(ctx, "failed to write cached value to datastore, err: %s", err.Error())
 				return handler.UnknownTransition, err
 			}
+			if r.GetDeckPath() != nil {
+				metadata, err := nCtx.DataStore().Head(ctx, *r.GetDeckPath())
+				if err == nil && metadata.Exists() {
+					tCtx.ow.GetDeckPath()
```

Comment:
Not needed, right?

Reply:
yeah, we can remove it if we add deck_uri to output info